gpiozero/output_devices.py

```diff
@@ -897,3 +897,92 @@ def stop(self):
         """
         self.forward_device.off()
         self.backward_device.off()
+
+
+class PhaseEnableMotor(SourceMixin, CompositeDevice):
+    """
+    Extends :class:`CompositeDevice` and represents a generic motor connected
+    to a Phase/Enable motor driver circuit.
+    The following code will make the motor turn "forwards"::
+        from gpiozero import PhaseEnableMotor
+        motor = PhaseEnableMotor(12, 5)
+        motor.forward()
+    :param int power:
+        The GPIO pin that the power input (PWM) of the motor driver chip is
+        connected to.
+    :param int direction:
+        The GPIO pin that the direction input of the motor driver chip is
+        connected to.
+    """
+    def __init__(self, power=None, direction=None):
+        if not all([power, direction]):
+            raise GPIOPinMissing(
+                'power and direction pins must be provided'
+            )
+        super(PhaseEnableMotor, self).__init__(
+            power_device = PWMOutputDevice(power),
+            direction_device = OutputDevice(direction),
+            _order = ('power_device', 'direction_device'))
+
+    @property
+    def value(self):
+        """
+        Represents the speed of the motor as a floating point value between -1
+        (full speed backward) and 1 (full speed forward).
+        """
+        return self.power_device.value if self.direction_device.is_active() else -self.power_device.value
```

Comment:
Yeah, `is_active` is a property rather than a method, so you don't want parentheses.
(decorators are another thing it takes a while to get your head around at first)